dronegen/gha.go

```diff
@@ -14,27 +14,48 @@
 
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"strings"
+	"time"
+)
 
 type ghaBuildType struct {
 	buildType
 	trigger
-	namePrefix      string
-	uploadArtifacts bool
-	srcRefVar       string
-	workflowRefVar  string
-	slackOnError    bool
+	pipelineName string
+	ghaWorkflow  string
+	srcRefVar    string
+	workflowRef  string
+	timeout      time.Duration
+	slackOnError bool
+	dependsOn    []string
+	inputs       map[string]string
 }
 
 func ghaBuildPipeline(b ghaBuildType) pipeline {
-	p := newKubePipeline(fmt.Sprintf("%sbuild-%s-%s", b.namePrefix, b.os, b.arch))
+	p := newKubePipeline(b.pipelineName)
 	p.Trigger = b.trigger
 	p.Workspace = workspace{Path: "/go"}
-	p.Environment = map[string]value{
-		"BUILDBOX_VERSION": buildboxVersion,
-		"RUNTIME":          goRuntime,
-		"UID":              {raw: "1000"},
-		"GID":              {raw: "1000"},
+	p.DependsOn = append(p.DependsOn, b.dependsOn...)
+
+	var cmd strings.Builder
+	cmd.WriteString(`go run ./cmd/gh-trigger-workflow `)
+	cmd.WriteString(`-owner ${DRONE_REPO_OWNER} `)
+	cmd.WriteString(`-repo teleport.e `)
+	cmd.WriteString(`-tag-workflow `)
+	fmt.Fprintf(&cmd, `-timeout %s `, b.timeout.String())
```

Comment:
Did you run `make dronegen`? I see the timeout setting being added here but I don't see it anywhere in the .drone.yml in this changeset.

Reply:
Oops. Good catch. Fixed.